addons/custom-zoom/addon.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "Custom Zoom Settings",
+  "description": "Adjust the minimum, maximum, speed, and start size of the zoom in the project editor, and autohide the controls.",
+  "tags": ["editor", "theme"],
+  "credits": [
+    {
+      "name": "ErrorGamer2000",
+      "link": "https://scratch.mit.edu/users/ErrorGamer2000/"
+    }
+  ],
+  "enabledByDefault": false,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "info": [
+    {
+      "type": "warning",
+      "text": "Making the blocks very small may injure your eyes. Be careful to avoid straining your eyes while using this addon.",
+      "id": "sizeWarning"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Maximum Zoom (100-500%)",
+      "id": "maxZoom",
+      "type": "integer",
+      "min": 50,
+      "default": 300,
+      "max": 500
+    },
+    {
+      "name": "Minimum Zoom (1-100%)",
+      "id": "minZoom",
+      "type": "integer",
+      "min": 1,
+      "default": 30,
+      "max": 100
+    },
```

Comment:
should we even have these as settings

Reply:
The minimum?

Reply:
So just set the max and min to values that can't be changed?